packages/core/src/events.ts

```diff
@@ -28,17 +29,20 @@ export interface CallEvent<T> extends Event<T> {
   data: string;
 }
 
-export interface ValueTransferEvent<S, T> extends Event<S> {
+export interface ValueTransferEvent<S, T, U> extends Event<S> {
   amount: BN;
   coinSymbol: string;
   from: string;
   to: string;
   transferType: T;
+  walletName: string;
+  walletType: U;
 }
 
 export type EthValueTransferEvent = ValueTransferEvent<
   EventStatus,
-  ValueTransferType
+  ValueTransferType,
+  WalletType
```

Comment:
Why are they generic type?

Reply:
`ValueTransferType` was used to be applied to `BtcValueTransferEvent` and `EthValueTransferEvent`. 
But, `BtcValueTransferEvent` has removed now!
So, I will remove generic for `ValueTransferType` and `WalletType` then use them directly.